pkg/cmd/cli/cmd/export.go

```diff
@@ -92,9 +94,16 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 	if err != nil {
 		return err
 	}
-	outputVersion, err := kcmdutil.OutputVersion(cmd, clientConfig.GroupVersion)
-	if err != nil {
-		return err
+
+	var outputVersion schema.GroupVersion
+	outputVersionString := kcmdutil.GetFlagString(cmd, "output-version")
```

Comment:
> No, output version must remain for generators, because generators have to work across multiple servers.

The `kubectl` stance was that nothing converts except for `kubectl convert`.  That makes it sound to me like the generator makes whatever it wants and `kubectl convert` is used to convert it to something else if you need/want to.